models/issue_list.go

```diff
@@ -429,6 +429,7 @@ func (issues IssueList) loadTotalTrackedTimes(e Engine) (err error) {
 
 		// select issue_id, sum(time) from tracked_time where issue_id in (<issue ids in current page>) group by issue_id
 		rows, err := e.Table("tracked_time").
+			Where(builder.Eq{"deleted": false}).
```

Comment:
So looking through Gitea I think this is more standard:

```suggestion
			Where("deleted= ?", false).
```